### PR TITLE
Updated styling in profile page

### PR DESCRIPTION
**Pull request by JC4github**
### Changes

Describe the Pull Request here. Add any references and info to help reviewers understand your changes.

- Changes to the styling in profile page

### Tests applied

What testing has been applied to validate changes are accurate and correct

- Visual inspection and screen resizing

### Issue ticket number(s)

Enter the issue numbers resolved by this pull request

- N/A

### Checklist

- [ ] Documented
- [ ] Linting tests successful
- [ ] merge updated prod branch into development branch

### CODE REVIEW

File: app/pages/profile/page.tsx [hunk 1]
```diff
@@ -34,42 +34,42 @@ export default function Profile() {
...
 
   return (
-    <div className="flex flex-col">
-      <div className="mt-12 text-center">
+    <div className="flex flex-col mb-12">
+      <div className="flex flex-row justify-between mt-6 text-left ml-12">
         <p className="font-bold text-3xl">Welcome to your profile page.</p>
+        <div className="mr-12">
+          <LogoutButton />
+        </div>
```
Comment (jaskfla) [inline app/pages/profile/page.tsx:42]: While we’re moving the logout button anyway: Does it need to be wrapped in a `<div>`?

I’m not sure what the `mr-12` class does (I can dig through the TSX and CSS files if necessary), but chances are we could just add the class to the root element in `LogoutButton`.

Shout out if this is a terrible idea!

**Review by jaskfla — APPROVED** [review]
Looks solid; I’ve just left one comment for a _potential_ minor refactor.

We may in future wish to parameterise the star colours, but I think at this point it’s not necessary.

File: app/pages/profile/page.tsx (hunk 1 above)
Comment (JC4github) [reply app/pages/profile/page.tsx:42]: this is tailwind css, it means margin-right 12

Comment (jaskfla) [reply app/pages/profile/page.tsx:42]: Ah, gotcha, thanks for clarifying!

**Pull request merged**